evap/results/templates/distribution_widget.html

```diff
@@ -2,13 +2,16 @@
 {% load evaluation_filters %}
 
 {% spaceless %}
-{% if average is not None %}
-    <div class="d-flex">
-        {% include "distribution_with_grade.html" with distribution=distribution average=average weight=weight weight_sum=weight_sum %}
+{% if course_or_evaluation.avg_grade is not None %}
+    <div class="d-flex"
+        data-toggle="tooltip" data-placement="left" data-fallback-placement='["left", "bottom"]'
+        title="{% spaceless %}{% include 'result_widget_tooltip.html' with question_result=course_or_evaluation.single_result_rating_result weight_info=course_or_evaluation|weight_info %}{% endspaceless %}"
+    >
+        {% include "distribution_with_grade.html" with distribution=course_or_evaluation.distribution average=course_or_evaluation.avg_grade weight_info=course_or_evaluation|weight_info %}
     </div>
 {% else %}
     <div class="d-flex"{% if not question_result %} data-toggle="tooltip" data-placement="left" title="{% trans 'Not enough answers were given.' %}"{% endif %}>
-        {% include "distribution_with_grade_disabled.html" with icon="far fa-eye-slash" %}
+        {% include "distribution_with_grade_disabled.html" with icon="far fa-eye-slash" %} {# QUESTION: Do we want weight_info here? #}
```

Comment:
I was not quite sure how to handle cases where some weighted evaluations are included in the result and some are excluded because answers are missing. In this case, I think the numbers are a bit wrong.
I remember that I discussed with @janno42 this problem for a minute, but I can’t remember possible solutions.

Reply:
we thought that just keeping the numbers although they don't add up to 100% is okay.